Gemfile

```diff
@@ -3,14 +3,12 @@ source "https://rubygems.org"
 gem 'rake'
 
 group :test do
-  gem 'rubyunit'
-  gem "codeclimate-test-reporter", require: false
-  gem 'coveralls', require: false
-  gem 'scrutinizer-ocular', require: false
+  gem 'rspec'
```

Comment:
We're going to want to keep these three lines, as we want to leverage these services just as soon as we get everything using Travis-CI.

```ruby
gem "codeclimate-test-reporter", require: false
gem 'coveralls', require: false
gem 'scrutinizer-ocular', require: false
```